test/test_model.jl

```diff
@@ -1216,4 +1216,32 @@ function test_show_variable_not_owned()
     return
 end
 
+function test_direct_mps_model()
+    model = direct_model(MOI.FileFormats.MPS.Model())
+    @test occursin("unknown", sprint(show, model))
+    @variable(model, x >= 0)
+    io = IOBuffer()
+    write(io, backend(model))
```

Comment:
This is one potential gotcha: `write_to_file(model, "filename")` still involves a copy because we don't/can't check if the `backend(model)` implements an efficient `write`.

Reply:
I think this is fine, if you're using `direct_model` you can also look up the slightly faster way to write the model.